data-node/src/test/java/org/graylog/datanode/periodicals/NodePingPeriodicalTest.java

```diff
@@ -81,4 +90,38 @@ void doRunWithRegister() throws NodeNotFoundException {
                 Mockito.any()
```

Comment:
If you change that to match "localhost" then the new test becomes redundant

Reply:
thx. I made the test more meaningful and removed the new test.